src/igvfd/schemas/signal_file.json

```diff
@@ -0,0 +1,106 @@
+{
+    "title": "Signal File",
+    "$id": "/profiles/signal_file.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submission of a file containing signal data.",
+    "required": [
+        "award",
+        "lab",
+        "md5sum",
+        "file_format",
+        "file_set",
+        "content_type",
+        "reference_files",
+        "strand_specificity"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases",
+        "md5sum"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "file.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/processed_files"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "file.json#/dependentSchemas"
+        ]
+    },
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "content_type": {
+            "comment": "Content Type describes the content of the file.",
+            "enum": [
+                "signal",
+                "signal of all reads",
+                "signal of unique reads",
+                "signal p-value",
+                "raw signal",
+                "read-depth signal",
+                "control signal",
+                "fold over change control"
+            ]
+        },
+        "file_format": {
+            "enum": [
+                "bigWig"
+            ]
+        },
+        "strand_specificity": {
+            "title": "Strand Specificity",
+            "description": "The strandedness of the signal file: plus, minus, or unstranded.",
+            "type": "string",
+            "enum": [
+                "plus",
+                "minus",
+                "unstranded"
+            ]
+        },
+        "filtered": {
+            "title": "Filtered",
+            "description": "Indicates if the signal file is filtered.",
+            "type": "boolean"
+        },
+        "normalized": {
+            "title": "Normalized",
+            "description": "Indicates if the signal file is normalized.",
+            "type": "boolean"
+        },
+        "start_view_position": {
+            "title": "Start View Position",
+            "description": "The starting position within the chromosome for viewing the signal.",
```

Comment:
We should probably mention that this is supposed to be a 0-based coordinate. Maybe "The 0-based coordinate for the default starting position when viewing the signal in a genome browser"?

Reply:
added